mkosi/__init__.py

```diff
@@ -2142,6 +2147,34 @@ def make_uki(
         extract_pe_section(context, output, ".initrd", context.staging / context.config.output_split_initrd)
 
 
+def make_initrd_addon(context: Context, stub: Path, output: Path) -> None:
+    make_cpio(context.root, context.workspace / "initrd", sandbox=context.sandbox)
+    maybe_compress(
+        context,
+        context.config.compress_output,
+        context.workspace / "initrd",
+        context.workspace / "initrd",
+    )
+    arguments: list[PathString] = [
+        "--initrd",
+        workdir(context.workspace / "initrd"),
+    ]
+    options: list[PathString] = [
+        "--ro-bind",
+        context.workspace / "initrd",
+        workdir(context.workspace / "initrd"),
+    ]
```

Comment:
Nit: since they are arguments for the options
```suggestion
    arguments: list[PathString] = [
        "--initrd", workdir(context.workspace / "initrd"),
    ]  # fmt: skip
    options: list[PathString] = [
        "--ro-bind", context.workspace / "initrd",  workdir(context.workspace / "initrd"),
    ]  # fmt: skip
```